src/CommonITILObject.php

```diff
@@ -2728,7 +2728,7 @@ public function post_clone($source, $history)
 
     public function getCloneRelations(): array
     {
-        return [];
+        return [$this->userlinkclass, $this->grouplinkclass, $this->supplierlinkclass];
```

Comment:
Plugins implementations of `CommonITILObject` may not define some of these relations.
```suggestion
        $relations = [];
        
        if (is_a($this->userlinkclass, CommonITILActor::class, true)) {
            $relations[] = $this->userlinkclass;
        }
        if (is_a($this->grouplinkclass, CommonITILActor::class, true)) {
            $relations[] = $this->grouplinkclass;
        }
        if (is_a($this->supplierlinkclass, CommonITILActor::class, true)) {
            $relations[] = $this->supplierlinkclass;
        }
        
        return $relations;
```